imports/api/PointCalculationUtils.js

```diff
@@ -0,0 +1,132 @@
+import { Constants } from './Constants';
+import { NewGameUtils } from './NewGameUtils';
+
+export var PointCalculationUtils = {
+	jpn_dealin_delta,
+	jpn_selfdraw_delta,
+	jpn_mistake_delta
```

Comment:
Hmm, is it possible to keep the jpn object in PointCalculationUtils with members "dealin_delta = jpn_dealin_delta"?  It seems you removed it this entry in favour of just the suffix.